packages/did/src/DidBatcher/FullDidUpdateBuilder.ts

```diff
@@ -548,60 +516,42 @@ export class FullDidUpdateBuilder extends FullDidBuilder {
    *
    * @returns The [[SubmittableExtrinsic]] containing the details of a DID update with the provided details.
    */
+  // TODO: Remove ignore when we can test the consume function
+  /* istanbul ignore next */
   public async consume(
     signer: KeystoreSigner,
     submitter: IIdentity['address'],
     atomic = true
   ): Promise<SubmittableExtrinsic> {
-    const first: Extrinsic = atomic
-      ? this.apiObject.tx.utility.batchAll(this.firstBatch)
-      : this.apiObject.tx.utility.batch(this.firstBatch)
+    const batchFunction = atomic
+      ? this.apiObject.tx.utility.batchAll
+      : this.apiObject.tx.utility.batch
+
+    const batchLength = this.batch.length
```

Comment:
if you do assign this to a variable, you may as well use it everywhere ;)

Reply:
OMG who wrote this code? 😱😱😱 https://github.com/KILTprotocol/sdk-js/pull/473/commits/24ac0c56e2e9d17c141cf05adf67223a20a6e9e8